src/libraries/Microsoft.PowerFx.Core/Public/DisplayNames/SingleSourceDisplayNameProvider.cs

```diff
@@ -77,5 +77,24 @@ public override bool TryGetDisplayName(DName logicalName, out DName displayName)
         {
             return _logicalToDisplay.TryGetValue(logicalName, out displayName);
         }
+
+        public DisplayNameProvider RemoveField(DName lookupName)
```

Comment:
Why return DisplayNameProvider and not SingleSourceDisplayNameProvider?

Reply:
you are correct! Thanks!